### DISCUSSION TITLE
Sticky header in table

### DISCUSSION THREAD

**Discussion by matejdostal** [mantinedev/mantine#1828] [Q&A] [answered]
Would it be possible to add prop to Table component to enable sticky/fixed headers?

**Comment by rtivital** [comment, marked as answer]
No, you can do that by adding `position: sticky; top: 0;` styles to thead element, see example – https://ui.mantine.dev/component/table-scroll-area

**Reply by matejdostal** [reply]
Oh, that's great. Thank you.
